packages/plugin-npm-cli/sources/commands/npm/login.ts

```diff
@@ -0,0 +1,124 @@
+import {Configuration, Ident, MessageName}              from '@berry/core';
+import {PluginConfiguration, StreamReport, structUtils} from '@berry/core';
+import {npmHttpUtils}                                   from '@berry/plugin-npm';
+import inquirer                                         from 'inquirer';
+import {Readable, Writable}                             from 'stream';
+
+const SUCCESS_MESSAGE = `Successfully logged in`;
+const ERROR_MESSAGE = `Invalid Authentication`;
+
+// eslint-disable-next-line arca/no-default-export
+export default (clipanion: any, pluginConfiguration: PluginConfiguration) => clipanion
+
+  .command(`npm login [-s,--scope SCOPE]`)
+  .categorize(`Npm-related commands`)
+  .describe(`login to registry`)
+
+  .detail(`
+    This command will ask you for your username, password, and 2FA One-Time Password (when it applies). It will then modify your local configuration (in your home folder) to reference the new tokens thus generated.
+
+    Adding the \`-s,--scope\` flag will cause the authentication to be done against whatever registry is configured for the associated scope (see also \`npmScopes\`).
+  `)
+
+  .example(
+    `Login to the default registry`,
+    `yarn npm login`,
+  )
+
+  .example(
+    `Login to a scoped registry`,
+    `yarn npm login --scope private`,
```

Comment:
I would have used something else than `private` (for example `my-scope`) to underline that the value of the `scope` property doesn't have a meaning by itself.